StudyBuddy/Managers/UserManager.cs

```diff
@@ -32,8 +37,69 @@ public UserId RegisterUser(string username, UserFlags flags, UserTraits traits)
 
         User newUser = new(userId, username, flags, traits);
         s_users.Add(newUser);
+
         return userId;
     }
 
     public List<IUser> GetAllUsers() => s_users.Where(u => u != null).Cast<IUser>().ToList();
+
+
+    public void LoadUsersFromCsv(string filePath)
```

Comment:
Maybe look in to using [CsvHelper](https://joshclose.github.io/CsvHelper/getting-started/)?

Reply:
Additional fuckery would be unreal. May be better to leave it as it is. Since the method will be scrapped after DB implemented

Reply:
Please no swearing, keep it family friendly pg-13, thanks! 😄